modules/highgui/src/cap_ximea.cpp

```diff
@@ -16,7 +16,7 @@ class CvCaptureCAM_XIMEA : public CvCapture
 
     virtual bool open( int index );
     virtual void close();
-    virtual double getProperty(int);
+    virtual double getProperty(int) const;
```

Comment:
Base class doesn't use `const` in 2.4: https://github.com/Itseez/opencv/blob/2.4/modules/highgui/src/precomp.hpp
Does it work really?